src/Files.App/UserControls/Pane/ShelfPane.xaml.cs

```diff
@@ -86,13 +86,13 @@ private void ListView_DragItemsStarting(object sender, DragItemsStartingEventArg
 			dataObjectProvider.SetDataObject(ppDataObject);
 		}
 
-		public IList<ShelfItem>? ItemsSource
```

Comment:
This does not need to be an `ObservableCollection`

Reply:
IList doesn't seem to update the image visibility when the list count changes.